R/model.R

```diff
@@ -229,8 +239,9 @@ enw_inits <- function(data) {
 #' @param verbose Logical, defaults to `TRUE`. Should verbose
 #' messages be shown.
 #'
-#' @param stanc_options A list of options to pass to the `stanc_options` of [cmdstanr::cmdstan_model()]
-#' by default "01" is passed which specifies simple optimisations should be done by the prior to compilation
+#' @param stanc_options A list of options to pass to the `stanc_options` of
+#' [cmdstanr::cmdstan_model()] by default "01" is passed which specifies simple
+#' optimisations should be done by the prior to compilation
```

Comment:
#' @param stanc_options A list of options to pass to the `stanc_options` of
#' [cmdstanr::cmdstan_model()]**. By** default "01" is passed, which specifies simple
#' optimisations should be done by **cdmstan** prior to compilation.